oracle-price-monitor/src/autotask.js

```diff
@@ -0,0 +1,270 @@
+// ORACLE PRICE ALERT - note there's never been an alert to this agent yet
+// agent id - 0x8a6e2ae3b279c561f7ba64e5011313df19e08b1f398d59d94dfbd4148e6cafce
+
+const axios = require('axios');
+const ethers = require('ethers');
+const needle = require('needle');
+
+// import the DefenderRelayProvider to interact with its JSON-RPC endpoint
+const { DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
+
+const TOKEN_ABI = [
+  'function decimals() view returns (uint8)',
+  'function symbol() view returns (string)',
+];
+const MAKER_TOKEN_ABI = [
+  'function decimals() view returns (uint256)',
+  'function symbol() view returns (bytes32)',
+];
+const CTOKEN_ABI = ['function underlying() view returns (address)'];
+
+const makerTokenAddress = '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'.toLowerCase();
+const saiTokenAddress = '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'.toLowerCase();
+const oddTokens = [makerTokenAddress, saiTokenAddress];
+
+const fortaApiEndpoint = 'https://api.forta.network/graphql';
+
+async function getDecimalsAndSymbol(cTokenAddress, provider) {
+  const cTokenContract = new ethers.Contract(
+    cTokenAddress,
+    CTOKEN_ABI,
+    provider,
+  );
+  const underlyingTokenAddress = await cTokenContract.underlying();
+
+  let decimals;
+  let symbol;
+  if (oddTokens.indexOf(underlyingTokenAddress.toLowerCase()) !== -1) {
+    const underlyingTokenContract = new ethers.Contract(
+      underlyingTokenAddress,
+      MAKER_TOKEN_ABI,
+      provider,
+    );
+
+    decimals = await underlyingTokenContract.decimals();
+    // need to convert decimals from uint256 to uint8
+    decimals = parseInt(decimals.toString(), 10);
+
+    symbol = await underlyingTokenContract.symbol();
+    // need to convert symbol from bytes32 to string
+    symbol = ethers.utils.parseBytes32String(symbol);
+  } else {
+    const underlyingTokenContract = new ethers.Contract(
+      underlyingTokenAddress,
+      TOKEN_ABI,
+      provider,
+    );
+    decimals = await underlyingTokenContract.decimals();
+    symbol = await underlyingTokenContract.symbol();
+  }
+  return { decimals, symbol };
+}
+
+function formatAmountString(amount, decimals) {
+  console.log("amount here", amount)
```

Comment:
```suggestion
  console.log('amount here', amount)
```
Ref: https://github.com/airbnb/javascript#strings--quotes